scheduler/pkg/kafka/pipeline/multi_topic_consumer.go

```diff
@@ -56,13 +58,17 @@ func NewMultiTopicsKafkaConsumer(
 		requests: cmap.New(),
 		tracer:   tracer,
 	}
-	err := consumer.createConsumer()
+	err := consumer.createConsumer(logger)
 	return consumer, err
 }
 
-func (c *MultiTopicsKafkaConsumer) createConsumer() error {
+func (c *MultiTopicsKafkaConsumer) createConsumer(logger log.FieldLogger) error {
 	consumerConfig := kafka_config.CloneKafkaConfigMap(c.config.Consumer)
 	consumerConfig["group.id"] = c.id
+	consumerConfig["go.events.channel.enable"] = true
```

Comment:
This channel should not be enabled at the same time with the Poll() method. There will be cases where the Poll simply looses events if this setting is on (because the corresponding events were sucked into the `events.channel`)